P5/Source/Guidelines/en/TD-DocumentationElements.xml

```diff
@@ -634,18 +634,20 @@ to mark any technical term, thus:
         <specDesc key="att.repeatable" atts="minOccurs maxOccurs"/>
       </specList>
       </p>
-      <p>Additionally, two wrapper elements are provided to indicate
-      whether the components listed as their children form a sequence
-      or an alternation:
+      <p>Additionally, three wrapper elements are provided to indicate
+      whether the components listed as their children form a sequence,
+      an alternation, or are interleaved:
       <specList>
         <specDesc key="sequence"/>
         <specDesc key="alternate"/>
+        <specDesc key="interleave"/>
       </specList>
-      These two wrapper elements are also members of <ident
+      These three wrapper elements are also members of <ident
       type="class">att.repeatable</ident>. References listed as
       children of <gi>sequence</gi> must appear in the order and
-      cardinality specified. Only one of the references listed as
-      children of <gi>alternate</gi> may appear, although the
+      cardinality specified whereas references listed as children
+      of <gi>interleave</gi> may appear in any order. Only one of the 
```

Comment:
Can-of-worms department.

(Note to @joeytakeda — most of the problem here is not from your improvement, but has been here all along and is just foregrounded by your change.)

This sentence does not mention that cardinality of the stuff inside `<interleave>`. But on thinking of how to re-word it, I realize that these sentences discuss the cardinality of _references_, when what we are talking about is the cardinality of _the thing referenced_. I am not sure what, if anything, we should do about this.